packages/parser-sample/src/grammar-validation.ts

```
@@ -0,0 +1,173 @@
+#!/usr/bin/env bun
+/**
+ * Grammar Validation Script
+ * 
+ * Tests all sample grammars to ensure they parse correctly
+ * and validates the generated parsers work with sample data.
+ */
+
+import { grammarDefinition, generateTypeScriptParser } from "tpeg-parser";
+import { readdir, readFile } from "fs/promises";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)